backend/no_waste.py

```diff
@@ -25,36 +25,58 @@ class Recipe(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     recipe_name = db.Column(db.String(255))
     recipe = db.Column(db.Text)
+    ingredients = db.Column(db.Text)
+
+    @property
+    def serialize(self):
+        return {
+            "title": self.recipe_name,
+            "imgUrl": 'https://loremflickr.com/320/240/food',
+            "description": self.recipe,
+            "ingredients": self.ingredients.split(',')
+        }
+
+
+class Product(db.Model):
+    __tablename__ = "products"
+    id = db.Column(db.Integer, primary_key=True)
+    product_name = db.Column(db.String(255))
+
 
 
 db.create_all()
 
 if Recipe.query.count() == 0:
-    # insert initial state
-    recipe = Recipe(id=1, recipe_name="Kielbasa", recipe="Przepis na kielbase")
-    recipe2 = Recipe(id=2, recipe_name="Chleb", recipe="Przepis na chleb")
-    recipe3 = Recipe(id=3, recipe_name="Piwo", recipe="Przepis na piwo")
-    db.session.add(recipe)
-    db.session.add(recipe2)
-    db.session.add(recipe3)
+    recipes = [
+        Recipe(id=1, recipe_name="Kielbasa", recipe="Przepis na kielbase", ingredients="Kielbasa, Chleb, Ketchup"),
+        Recipe(id=2, recipe_name="Chleb", recipe="Przepis na chleb", ingredients="Drzdze, maka, olej"),
+        Recipe(id=3, recipe_name="Piwo", recipe="Przepis na piwo", ingredients="drozdze, chmiel"),
+        Product(id=1, product_name="Ziemniak"),
+        Product(id=2, product_name="Kielbasa"),
+        Product(id=3, product_name="Pomidor"),
+    ]
+
+    db.session.add_all(recipes)
     db.session.commit()
 
 
 # all recipes in database
 @app.route("/recipes")
 def recipelist():
-    return jsonify([recipes.recipe_name for recipes in Recipe.query.all()])
+    return jsonify([x.serialize for x in Recipe.query.all()])
```

Comment:
why x?

Reply:
Kuba recommended using serialize in both requests.
Beside that i thought that I should return all not only name

Reply:
I ask why you use name x. Recipes was good name.